### PR TITLE
Main-section desktop design

### PR DESCRIPTION
**Pull request by ehs9nino**
Now I want to make the design compatible with my desktop version.

### CODE REVIEW

**Comment by ehs9nino** [comment]
I have tried to make a responsive web design & applied media queries. For 786px or above is showing a desktop compatible version. I try to make it more similar to the design that I am following from Figma.
(link- https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A1278)

Please let me know If I need any changes making a design similar to that or maybe if even I am not applying any important concepts, thanks!

File: desktop.css [hunk 1]
```diff
@@ -0,0 +1,230 @@
+/* stylelint-disable csstree/validator */
```
Comment (thecodechaser) [inline desktop.css:1]: - [ ] There is some margin around the page body and a white border appearing and it's not looking good, Please fix this issue to make your project looks more professional.
 - To fix this you can add `margin: 0;` in your body class.

File: index.html [hunk 2]
```diff
@@ -43,114 +62,127 @@ <h2>I`am a Software Developer</h2>
         <div class="work1-visual">
           <h1>My Recent Works</h1>
           <div class="line"></div>
```
Comment (thecodechaser) [inline index.html]: - [ ] Great job on the design but for this project you are supposed to stick to the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)design as much as possible. Please reduce the gap between project cards and make sure it matches the design on the template.

### Your design

image

### Figma design

image

File: index.html [hunk 3]
```diff
@@ -43,114 +62,127 @@ <h2>I`am a Software Developer</h2>
         <div class="work1-visual">
           <h1>My Recent Works</h1>
```
Comment (thecodechaser) [inline index.html]: - [ ] When we hover on any project the content of the card should be replaced with the `see project` button and the background without gradient. Please fix this issue and make sure it matches the design on the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)template.

#### See this image of Figma design, The mouse hovers on the first project and we can see it's showing the see project button and the content of the card has been changed. If you still need more clarification please see this [project](https://devmahmoud.github.io/portfolio/).

image

File: index.html [hunk 4]
```diff
@@ -165,69 +197,78 @@ <h1>About me</h1>
         </p>
         <div class="about-me-btn"><input type="button" class="about-me-button" value="Get my Resume"></div>
```
Comment (thecodechaser) [inline index.html:198]: - [ ] Please apply the correct padding for this button and make sure it matches the design on the template.

### Your design

image

### Figma design

image

File: index.html [hunk 5]
```diff
@@ -165,69 +197,78 @@ <h1>About me</h1>
...
-      </ul>
-      <div class="topic">
-        <img src="./svg/frameworks.svg" alt="" />
-        <p>Frameworks</p>
+      <div class="grid1">
+        <div class="topic">
+          <img src="./svg/language.svg" alt="" />
+          <p>Language</p>
+        </div>
+        <ul class="discussion">
```
Comment (thecodechaser) [inline index.html:211]: - [ ] Please apply the correct gap between the technologies, each technology should be listed in the same line and make sure it matches the design on the template.

### Your design

image

### Figma design

image

File: index.html [hunk 6]
```diff
@@ -165,69 +197,78 @@ <h1>About me</h1>
...
-        <li class="form-button">
-          <button class="submit" type="submit">Get In Touch</button>
-        </li>
-      </ul>
-    </form>
+    <div class="form-grid">
+      <h2>
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </h2>
+      <form class="from1" action="https://formspree.io/f/xdoblzqe" method="post">
```
Comment (thecodechaser) [inline index.html:251]: - [ ] The form details contain approx 35% of the window width and the form contains the remaining width, Please make sure you follow the width for each section and your design should match the Figma template

### Your design

image

### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @ehs9nino👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (ehs9nino) [reply index.html]: @thecodechaser  thanks but here I am facing a problem that all the remaining pictures are coming with the gradients.  I understand the concept but If I know the way how to remove the gradient then I can easily apply the concept.
(https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A1278)
Please check is there any way I can pick a background picture from Figma without the gradient?

Comment (thecodechaser) [reply index.html]: @ehs9nino, Hello You can download the image from below :
 
Img Placeholder

Comment (ehs9nino) [reply index.html]: @thecodechaser  thanks but in that case, I will have only this image to use for all the backgrounds I guess.

Comment (thecodechaser) [reply index.html]: @ehs9nino, Yes you can use this image for all the backgrounds

Comment (ehs9nino) [reply index.html]: @thecodechaser  okay thanks!

File: desktop.css (hunk 1 above)
Comment (ehs9nino) [reply desktop.css:1]: @thecodechaser I can't understand the behavior or what exactly is happening when I try to inspect. for example, is showing outside from the body container. 

Please let me know if there is a recommendation.

Screenshot from 2022-04-01 15-31-48

**Comment by ehs9nino** [comment]
@thecodechaser I finalized this task according to the best of my understanding to make this desktop portfolio the best and also I try to stick to the Figma design as well. 
In addition, I did some major changes like I used media queries starting from 1024 pixels ( iPad Landscape & Desktops) instead of 768 pixels ( I iPad & Galaxy Tab). In order to show my content more responsive, I applied media queries additionally to some sections in my HTML document. 
Please guide me if there are any major changes required in this design, thanks!

**Review by OLIPLICHE — APPROVED** [review]
Hi @ehs9nino , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**